src/EventStore.Client/EventStore.Client.csproj

```diff
@@ -9,6 +9,7 @@
 		<PackageReference Include="Google.Protobuf" Version="3.14.0" />
 		<PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
 		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+		<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
**CONCERN:** Shouldn't we use `System.Text.Json` to not add additional dependency?